resources/views/activejourneys.blade.php

```diff
@@ -244,7 +244,7 @@ className: 'text-trwl text-center'
                                 "end": "{{ $event->end->format('Y-m-d') }}",
                                 "ts": {!! $event->getTrainStation() !!},
                                 "mapLink": "{{ route('statuses.byEvent', ['eventSlug' => $event->slug]) }}",
-                                "closestLink": `{!! stationLink($event->getTrainstation()->name) !!}`
+                                "closestLink": `<a href="{{route('trains.stationboard', ['provider' => 'train', 'station' => $event->getTrainstation()->id])}}" class="text-trwl clearfix">{{$event->getTrainstation()->name}}</a>`
```

Comment:
```suggestion
                                "closestLink": "<a href=\"{{route('trains.stationboard', ['provider' => 'train', 'station' => $event->getTrainstation()->id])}}\" class=\"text-trwl clearfix\">{{$event->getTrainstation()->name}}</a>"
```

Please check if this works. Might have made a typo